client/log_client.go

```diff
@@ -357,6 +357,24 @@ func (c *LogClient) AddSequencedLeaf(ctx context.Context, data []byte, index int
 	return err
 }
 
+// AddSequencedLeaves adds any number of pre-sequenced leaves to the log.
+func (c *LogClient) AddSequencedLeaves(ctx context.Context, dataByIndex map[int64][]byte) error {
```

Comment:
Does it not allow setting non-data fields of leaves?

Reply:
This API does not, just like `AddSequencedLeaf`.  We can revisit the API more generally in another PR if you'd like

Reply:
LGTM as long as it's consistent. We can change it when there is a case for it.